app/src/components/attachments/list/AttachmentsListItemMenuButton.tsx

```diff
@@ -85,21 +85,21 @@ const AttachmentsListItemMenuButton = (props: IAttachmentsListItemMenuButtonProp
                 View Details
               </MenuItem>
             )}
-            {props.attachmentStatus === PublishStatus.UNSUBMITTED && (
```

Comment:
Wouldn't this still apply for non admin users?

Reply:
![image](https://github.com/bcgov/biohubbc/assets/35311998/17493a4d-6a09-4d91-9433-9646c1463455)
We have a separate button that is unreleased currently for request to resubmit/remove.
So pure delete seemed like it was a admin only button